### ISSUE TITLE
A11y_VisualStudioCodeClient_Welcome Screen_ Screen reader: Voiceover is announcing extra information like "1 list item" for 'open folder' button

### ISSUE THREAD

**Issue by BhanuPrakashGit** [microsoft/vscode#180112]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #A11ySev2; #Device; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Oct2022; WCAG1.3.1; #Voiceover; #Screenreader.

### Environment and OS details:
Visual Studio Code Insiders
Screenreader: Voiceover

### Repro Steps:

1. Open Visual studio code editor
2. Tab till "Help" menu bar option .
3. Using "down" arrow key to navigate to "Getting started" and hit enter.
4. Tab navigate till "Open a folder button".
5. Turn on Voiceover
6. Verify that Voiceover is announcing "list" for the button.

### Actual Result:
Voiceover is announcing "list" for the "Open a folder" button.

### Expected Result:
Voiceover should not announce "list" for "Open a folder" button.

### Attachment:
![197939351-0c1e842a-9460-42c7-ad0e-7e8a0a321b75](https://user-images.githubusercontent.com/104351566/232457059-41ccf411-a4ce-4593-ac3f-337b199b3a25.gif)

**Comment by BhanuPrakashGit** [comment]
Reference bug: https://github.com/Microsoft/vscode/issues/164695#issue-1423839509

**Comment by meganrogge** [comment]
I believe this is a voiceover issue. 

Please file against VoiceOver here https://bugs.webkit.org/

@isidorn pls correct me if i'm wrong

**Comment by isidorn** [comment]
Correct. This is a VoiceOver bug.
